tests/firsttests.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test, expect } from '@playwright/test';
+
+test('Test 1: Verify login with valid credentials', async ({ page }) => {
+    await page.goto('https://practicesoftwaretesting.com/auth/login');
+    await page.fill('#email', 'customer@practicesoftwaretesting.com');
+    await page.fill('#password', 'welcome01');
+    await page.click('[data-test="login-submit"]')
```

Comment:
You can add data-test in the https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute
and use as

```
await page.getByTestId('login-submit').click();
```